### PR TITLE
1.1.8 release notes

### PR DESCRIPTION
**Pull request by duderino**

### CODE REVIEW

**Comment by duderino** [comment]
@geeknoid do I need to also update data/args.yml on the release-1.1 branch?

File: content/boilerplates/notes/1.1.8.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+## Bug fixes
+
+- Fix `PASSTHROUGH` `DestinationRules` for CDS clusters ([Issue 13744](https://github.com/istio/istio/issues/13744)).
+- Make the `appVersion` and `version` fields in the Helm charts display the correct Istio version ([Issue 14290](https://github.com/istio/istio/issues/14290)).
+- Fix Mixer crash affecting both policy and telemetry servers ([Issue 14235](https://github.com/istio/istio/issues/14235)).
+- Fix multi-cluster issue where two pods in different clusters could not share the same IP address ([Issue 14066](https://github.com/istio/istio/issues/14066)).
```
Comment (geeknoid) [inline content/boilerplates/notes/1.1.8.md]: We spell multicluster with no -

**Comment by geeknoid** [comment]
Only one small correction.

You're really getting the hang of this.

English I mean.

**Comment by duderino** [comment]
> Only one small correction.
> 
> You're really getting the hang of this.
> 
> English I mean.

touché

**Pull request merged**